Source/GoBot/ViewDatabaseSchema.sql

```diff
@@ -197,6 +197,23 @@ CREATE TABLE branches (
     FOREIGN KEY ( branch )      REFERENCES messages( "msg_id" )
 );
 
+-- unified mentions
```

Comment:
we can remove the old tables then, right?

Reply:
Yeah, i think we can just remove the old tables. I was afraid to do that in this PR in case it broke things for people in testing. So i think the removal should come after others have tested it.

Reply:
I see, pushing such working branches as builds should be done with extreme caution and accompany a note about downgrades (using existing db data with older versions might throw errors for instance and could need a wipe of the application data through delete and reinstall)

Non the less we need to choose one of two strategies for the upgrades:

1. write (sqlite-)code that does a full migration to the new schema (like, copy existing data to new tables)
2. bump the `schemaVersion` of `ViewDatabase.swift` and trigger a full refill without any migration

Reply:
I think my mistake is i don't need to do a refill, just a migration. It just wasn't clear to me how to do it.